.config/systemd/user/get-and-send-to-graphite:minutely@.timer

```diff
@@ -5,5 +5,9 @@ Description=Run get-and-send-to-graphite@%i each minute
 # OnUnitActiveSec is broken: https://github.com/systemd/systemd/issues/6680
 OnCalendar=minutely
 
+# randomize start to avoid using too much ressources
+AccuracySec=1s
```

Comment:
According to the systemd documentation, this should be `1us`:

```
In order to optimally stretch timer events over a certain range of time, set AccuracySec=1us and RandomizedDelaySec= to some higher value.
```

https://www.freedesktop.org/software/systemd/man/systemd.timer.html#AccuracySec=

Reply:
by using `1s`, it divises the delay into 20 slots, which look good enough IMO.

from the doc (with reference to our conseil federal maybe?)
> To optimize power consumption, make sure to set this value as high as possible and as low as necessary.

`1us` looks too intensive, would 100ms be better (-> 200 slots)?

Reply:
OK, let's go with 100ms, which is a bit below the shortest runtime. Even though I think the server wakes up every 1ms anyway...

Reply:
alright, 100ms it is :)

btw, and if systemd actually forces the kernel to wakeup every ms, one more reason to ditch it: on tickless kernels (such as most kernel theses day), having a wakeup source for no needed events this frequent is very bad for power consumption
https://en.wikipedia.org/wiki/Tickless_kernel